packages/extension-koni-ui/src/Popup/Home/Staking/StakingMenu.tsx

```diff
@@ -40,11 +40,17 @@ const MANUAL_CLAIM_CHAINS = [
   'shiden'
 ];
 
+const MANUAL_COMPOUND_CHAINS = [
+  'turing',
+  'turingStaging'
+];
+
 function StakingMenu ({ bondedAmount, className, networkKey, nextWithdrawal, nextWithdrawalAmount, redeemable, showClaimRewardModal, showMenu, showWithdrawalModal, toggleMenu, unbondingStake }: Props): React.ReactElement<Props> {
   const stakingMenuRef = useRef(null);
   const navigate = useContext(ActionContext);
   const networkJson = useGetNetworkJson(networkKey);
   const showClaimButton = MANUAL_CLAIM_CHAINS.includes(networkKey);
+  const showCompoundButton = MANUAL_COMPOUND_CHAINS.includes(networkKey);
   const { currentAccount: { account } } = useSelector((state: RootState) => state);
```

Comment:
using `const account = useSelector((state: RootState) => state.currentAccount.account)` to optimize performance

Reply:
Done